VariableAnalysis/Sniffs/CodeAnalysis/VariableAnalysisSniff.php

```diff
@@ -195,14 +195,11 @@ protected function isGetDefinedVars(File $phpcsFile, $stackPtr) {
   }
 
   /**
-   * @param int|bool $currScope
+   * @param int $currScope
    *
    * @return string
    */
   protected function getScopeKey($currScope) {
-    if ($currScope === false) {
```

Comment:
I'd like to review the history of this and make sure that it's no longer necessary.

Reply:
Ok, this guard is extremely old and I think is just no longer necessary.